devise_security_extension.gemspec

```diff
@@ -20,7 +20,11 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.required_ruby_version = '>= 2.1.0'
 
-  s.add_runtime_dependency 'railties', '>= 3.2.6', '< 6.0'
+  if RUBY_VERSION >= '2.4'
```

Comment:
I wasn't sure if this was the best way to handle supporting both 2.4 and pre 2.4. Any suggestions @natebird?